{"timestamp":"2025-07-11T21:40:21.682770","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-11T21:40:21.683022","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/bank_etl_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-11T21:40:21.710732Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.710860Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.710917Z","level":"info","event":"Current task name:start_log","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.710966Z","level":"info","event":"Dag name:bank_data_etl_process","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.711007","level":"info","event":"Connection Retrieved 'postgres_bank_logs'","logger":"airflow.hooks.base"}
{"timestamp":"2025-07-11T21:40:21.737047","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowFailException","exc_value":"Не удалось залогировать старт: 'Connection' object has no attribute 'commit'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/decorator.py","lineno":251,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/bank_etl_dag.py","lineno":57,"name":"start_log"}],"is_group":false,"exceptions":[]},{"exc_type":"AttributeError","exc_value":"'Connection' object has no attribute 'commit'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/opt/airflow/dags/bank_etl_dag.py","lineno":52,"name":"start_log"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-11T21:40:21.741717Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.741807Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.741876Z","level":"info","event":"Task:<Task(_PythonDecoratedOperator): start_log>","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-11T21:40:21.741941Z","level":"info","event":"Failure caused by Не удалось залогировать старт: 'Connection' object has no attribute 'commit'","chan":"stdout","logger":"task"}
